misk/web/tabs/web-actions/src/ducks/webActions.ts

```diff
@@ -901,8 +901,12 @@ export const processMetadata = (webActionMetadata: IWebActionAPI[]): any =>
     .value()
 
 function* handleMetadata() {
+  const useTestData = false
```

Comment:
what's the `useTestData` used for?

Reply:
Testing the web-actions tab without a live running service